IkeTurtle/gamedle
# Initial Database Implementation and Display of GuessingObjects

What has been done?

- First implementation of the database for the app.
- Added some sample data into the database.
- Created a way to show GuessingObject pictures and questions on different URLs.

How to test?

- Visit /GameSet<int> to check if GuessingObject pictures and questions are 
displayed
